src/pages-and-resources/discussions/app-config-form/apps/shared/discussion-topics/TopicItem.jsx

```diff
@@ -124,15 +125,19 @@ const TopicItem = ({
               </Collapsible.Visible>
               <Collapsible.Visible whenOpen>
                 {getHeading(true)}
-                <div className="pr-4 border-right">
-                  <IconButton
-                    onClick={deleteDiscussionTopic}
-                    alt={intl.formatMessage(messages.deleteAltText)}
-                    src={Delete}
-                    iconAs={Icon}
-                    variant="dark"
-                  />
-                </div>
+                {
+                  isGeneralTopic && (
```

Comment:
Can you check this condition here? I do not think we want to show `delete` button for the `isGeneralTopic`?

Reply:
I updated it.